src/components/cms/BotBuilder/Preview/Preview.js

```diff
@@ -2,12 +2,182 @@ import React, { Component } from 'react';
+  max-width: 95%;
+  width: 330px;
+  margin: 10px auto auto auto;
+  padding: 0 20px;
+  @media (max-width: 520px) {
+    width: 100%;
+    padding: 0 10px;
+  }
+`;
+
+const PreviewChatContainer = styled.div`
+  min-height: 460px;
+  @media (max-width: 520px) {
+    width: 100%;
+  }
+`;
+
+const moveFromBottomFadeKeyframe = keyframes`
+0% {
+  opacity: .2;
+  transform: translateY(5%)
+}
+`;
+
+const SUSIFrameContainer = styled.div`
+  overflow: auto;
+  max-height: 610px;
+  min-height: 280px;
+  max-width: 100%;
+  margin: 0;
+  border-radius: 8px;
+  box-shadow: 0 5px 40px rgba(0, 0, 0, 0.16);
+  overflow-x: hidden;
+  height: 460px;
+  width: 100%;
+  animation: ${moveFromBottomFadeKeyframe} 0.3s ease both;
+
+  media(max-width: 667px) {
+    width: 100%;
+    left: 0;
+    right: 0;
+    top: 0;
+    border-radius: 0;
+    max-height: none;
+  }
+`;
+
+const SUSIFrameWrapper = styled.div`
+  position: relative;
+  background-repeat: no-repeat;
+  background-color: #fff;
+  height: 100%;
+  width: 100%;
+  background-position: 150px 200px;
+`;
+
+const SUSIMessageContainer = styled.div`
+  background-color: ${props => props.backgroundColor};
+  background-image: ${props => `url(${props.backgroundImage})`};
+`;
+
+const UserMessageContainer = styled.div`
+  :after {
+    content: '';
+    position: absolute;
+    box-sizing: border-box;
+    right: -10px;
+    top: 36px;
+    transform: rotate(-135deg);
+    border: 8px solid;
+    background-color: transparent;
+    transform-origin: 0 0;
+    border-color: ${props =>
+      `transparent transparent ${props.backgroundColor} ${props.backgroundColor}`};
+  }
+`;
+
+const CloseButton = styled(Close)`
+  z-index: 2147483001;
+  cursor: pointer;
+  position: absolute;
+  right: 15px;
+  top: 15px;
+  width: 30px;
+  height: 30px;
+  background-color: #666;
+  border-radius: 50%;
+`;
+
+const SendButtonWrapper = styled.pre`
+  margin: 0;
+  cursor: pointer;
+  font-family: Open Sans, sans-serif;
+  overflow: hidden;
+`;
+
+const launcherFrameAppearKeyframe = keyframes`
+0% {
+  opacity: 0;
+  -webkit-transform: scale(.5);
+  transform: scale(.5)
+}
+to {
+  opacity: 1;
+  -webkit-transform: scale(1);
+  transform: scale(1)
+}
+`;
+
+const SUSILauncherContainer = styled.div`
+  right: 0;
+  direction: ltr;
+  bottom: 15px;
+`;
+
+const SUSILauncherWrapper = styled.div`
+  width: 60px;
+  background-size: 60px;
+  text-align: right;
+  float: right;
+  margin-right: 0%;
+  margin-top: 15px;
+  font-family: Open Sans, sans-serif;
+  border-radius: 5em;
+  cursor: pointer;
+  transition: transform .15s ease-in-out, box-shadow .15s ease-in-out;
+  visibility: visible;
+  transform: translateY(150px);
+  animation: ${launcherFrameAppearKeyframe} .25s ease forwards
+  padding: 0;
+  height: auto;
+  :hover {
+    visibility: visible;
+    box-shadow: 0 4px 42px 0 rgba(0, 0, 0, .25);
+  }
+`;
+
+const SUSILauncherButton = styled.div`
+  background-color: ${props => props.backgroundColor};
```

Comment:
not required

Reply:
This is required as we have Avatar background option in BotBuilder --> Design --> Avatar background